_includes/song.vto

```diff
@@ -1,34 +1,64 @@
 ---
 layout: layouts/page.vto
 ---
-Creators: {{tags}}
+<link rel="stylesheet" href="../scoresly.css">
+<div class="content">
+  <div class="post-credits">
+    <div class="left-credits">
+        {{ if lyricist}}
+        <div>Lyricist: {{lyricist}}</div>
+        {{ /if }}
+    </div>
 
-{{ if composer}}
-Composer: {{composer}}
-{{ /if }}
 
-{{ if lyricist}}
-Lyricist: {{lyricist}}
-{{ /if }}
+    <div class="right-credits">
+      <div>Creators: {{tags}}</div>
 
-{{ if arranger}}
-Arranger: {{arranger}}
-{{ /if }}
+      {{ if composer}}
+      <div>Composer: {{composer}}</div>
+      {{ /if }}
 
+      {{ if arranger}}
+      <div>Arranger: {{arranger}}</div>
+      {{ /if }}
+    </div>
+  </div>
 
-<div id="osmdContainer"></div>
+  <div id="osmdContainer" ></div>
+</div>
 <script src="https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/releases/download/1.8.8/opensheetmusicdisplay.min.js"></script>
+
 <script>
-const osmd = new opensheetmusicdisplay.OpenSheetMusicDisplay("osmdContainer");
-osmd.setOptions({
-    backend: "svg",
-    drawTitle: false,
-    drawCredits: false,
-    scale: 0.8
-});
-
-osmd.load(`{{content}}`).then(function() {
+  function getScaleForScreenWidth() {
+    const windowSize = window.innerWidth;
+    if (windowSize < 800) {
+      return 0.5;
```

Comment:
```suggestion
      return 0.6;
```
Maybe this improves mobile vertical a little bit? Or 0.55?

Reply:
Let's compromise and do 0.55!